packages/api/es/search.js

```diff
@@ -35,10 +35,11 @@ const getCredentials = () =>
  * @returns {string} elasticsearch local address
  */
 const getLocalEsHost = () => {
-  const prot = (process.env.LOCAL_ES_HOST_PROTOCOL) ? process.env.LOCAL_ES_HOST_PROTOCOL : 'http';
-  if (process.env.LOCAL_ES_HOST) return `${prot}://${process.env.LOCAL_ES_HOST}:9200`;
-  if (process.env.LOCALSTACK_HOST) return `${prot}://${process.env.LOCALSTACK_HOST}:4571`;
-  return `${prot}://localhost:9200`;
+  const port = process.env.LOCAL_ES_HOST_PORT || 9200;
```

Comment:
Just a new local-only ENV var for choosing a port to find the elasticsearch when testing locally.